warehouse/templates/accounts/login.html

```diff
@@ -86,16 +82,19 @@ <h1 class="page-title">{% trans title=title %}Log in to {{ title }}{% endtrans %
             </ul>
             {% endif %}
           </div>
-        </div>
-
-        <div class="form-group">
-          <div class="split-layout split-layout--table">
-            <div>
-              <input type="submit" value="{% trans %}Log in{% endtrans %}" class="button button--primary">
+          <div class="form-group">
+            <div class="split-layout split-layout--table">
+              <div>
+                <input type="submit" value="{% trans %}Log in{% endtrans %}" class="button button--primary">
+              </div>
+              <label for="show-password">
+                <input data-action="change->password#togglePasswords" data-target="password.showPassword"
+                  id="show-password" type="checkbox">&nbsp;{% trans %}Show password{% endtrans %}
+              </label>
+              <span>
+                <a href="{{request.route_url('accounts.request-password-reset')}}">{% trans %}Forgot password?{% endtrans %}</a>
+              </span>
```

Comment:
I _think_ wrapping these in a `div` will cause them to both be moved to the right side, but stacked on top of each other

```suggestion
              <div>
                <label for="show-password">
                  <input data-action="change->password#togglePasswords" data-target="password.showPassword"
                    id="show-password" type="checkbox">&nbsp;{% trans %}Show password{% endtrans %}
                </label>
                <span>
                  <a href="{{request.route_url('accounts.request-password-reset')}}">{% trans %}Forgot password?{% endtrans %}</a>
                </span>
              </div>
```